.github/actions/common/build-target/action.yaml

```
@@ -44,13 +44,7 @@ runs:
         run: |
           mkdir build
           cd build
-          if [ "${{ inputs.multilib }}" == "multilib" ]; then
-            ../configure --prefix=$(pwd) --with-multilib-generator='${{ inputs.multilib_target_string }}'
-          else
-            TARGET_TUPLE=($(echo ${{ inputs.target }} | tr "-" "\n"))
-            ../configure --prefix=$(pwd) --with-multilib-generator="${TARGET_TUPLE[0]}-${TARGET_TUPLE[1]}--"
-          fi
-
+          ../../configure-scripts/${{ inputs.multilib_target_string }}
```

```suggestion
          ../../configure-scripts/${{ inputs.multilib_target_string }}

```
Whitespace got clobbered